### PR TITLE
replaced four invalid links to new links

### PR DESCRIPTION
**Pull request by RAJU2529**
as per the user report #4473 , so i replaced  four new links from  tech community blog, article was written by **John Marlin
John Marlin**  from Microsoft  

here are the new links 

**https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372005**
**https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372008**
**https://techcommunity.microsoft.com/t5/failover-clustering/site-aware-failover-clusters-in-windows-server-2016/ba-p/372060**
**https://techcommunity.microsoft.com/t5/failover-clustering/workgroup-and-multi-domain-clusters-in-windows-server-2016/ba-p/372059**

### CODE REVIEW

**Comment by RAJU2529** [comment]
This PR gave to me some trouble, after adding correct links, i got pain while adjusting space for links.
One link gone to the next line, I could not able to adjust to sit in correct place next to the topic

**Comment by illfated** [comment]
Looks OK to me, so I am curious which line you are referring to. If you look at the Rich diff view (https://github.com/MicrosoftDocs/windowsserverdocs/pull/4476/files?short_path=5c61912#diff-5c6191248129cd5e53f627dedd8455f9) you will see that the text part is unchanged. See examples of your new link URLs below:
- For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Cluster Log](https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372005).
- For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Active Dump](https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372008).
- For more information, see [Site-aware Failover Clusters in Windows Server 2016](https://techcommunity.microsoft.com/t5/failover-clustering/site-aware-failover-clusters-in-windows-server-2016/ba-p/372060).
- For more information, see [Workgroup and Multi-domain clusters in Windows Server 2016](https://techcommunity.microsoft.com/t5/failover-clustering/workgroup-and-multi-domain-clusters-in-windows-server-2016/ba-p/372059)

But yes, I understand that it may look somewhat discouraging to see the number of lines increase due to longer URLs and added line wraps. I just don't think it is worth worrying about.

Addendum: I will make a quick review, mostly to see if there are any whitespace changes related to your link URL changes. Be right back.

File: WindowsServerDocs/failover-clustering/whats-new-in-failover-clustering.md [hunk 1]
```diff
@@ -170,13 +170,13 @@ In Windows Server 2016, virtual machine storage resiliency is aware and optimize
 
 To help diagnose issues with failover clusters, Windows Server 2016 includes the following:  
 
-- Several enhancements to cluster log files (such as Time Zone Information and DiagnosticVerbose log) that makes is easier to troubleshoot failover clustering issues. For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Cluster Log](https://blogs.msdn.com/b/clustering/archive/2015/05/15/10614930.aspx). 
+- Several enhancements to cluster log files (such as Time Zone Information and DiagnosticVerbose log) that makes is easier to troubleshoot failover clustering issues. For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Cluster Log](https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372005).
```
Comment (illfated) [inline WindowsServerDocs/failover-clustering/whats-new-in-failover-clustering.md:173]: End-of-line whitespace (blanks) removal + typo correction:
```suggestion
- Several enhancements to cluster log files (such as Time Zone Information and DiagnosticVerbose log) that makes it easier to troubleshoot failover clustering issues. For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Cluster Log](https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372005).
```

File: WindowsServerDocs/failover-clustering/whats-new-in-failover-clustering.md [hunk 2]
```diff
@@ -170,13 +170,13 @@ In Windows Server 2016, virtual machine storage resiliency is aware and optimize
...
 
-- A new  a dump type of **Active memory dump**, which filters out most memory pages allocated to virtual machines, and therefore makes the memory.dmp much smaller and easier to save or copy. For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Active Dump](https://blogs.msdn.com/b/clustering/archive/2015/05/18/10615526.aspx). 
+- A new  a dump type of **Active memory dump**, which filters out most memory pages allocated to virtual machines, and therefore makes the memory.dmp much smaller and easier to save or copy. For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Active Dump](https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372008).
```
Comment (illfated) [inline WindowsServerDocs/failover-clustering/whats-new-in-failover-clustering.md:175]: End-of-line whitespace (blanks) removal:
```suggestion
- A new  a dump type of **Active memory dump**, which filters out most memory pages allocated to virtual machines, and therefore makes the memory.dmp much smaller and easier to save or copy. For more information, see [Windows Server 2016 Failover Cluster Troubleshooting Enhancements - Active Dump](https://techcommunity.microsoft.com/t5/failover-clustering/windows-server-2016-failover-cluster-troubleshooting/ba-p/372008).
```

File: WindowsServerDocs/failover-clustering/whats-new-in-failover-clustering.md [hunk 3]
```diff
@@ -170,13 +170,13 @@ In Windows Server 2016, virtual machine storage resiliency is aware and optimize
...
 
 ### <a name="BKMK_SiteAware"></a>Site-aware Failover Clusters
 
-Windows Server 2016 includes site- aware failover clusters that enable group nodes in stretched clusters based on their physical location (site). Cluster site-awareness enhances key operations during the cluster lifecycle such as failover behavior, placement policies, heartbeat between the nodes, and quorum behavior. For more information, see [Site-aware Failover Clusters in Windows Server 2016](https://blogs.msdn.com/b/clustering/archive/2015/08/19/10636304.aspx). 
+Windows Server 2016 includes site- aware failover clusters that enable group nodes in stretched clusters based on their physical location (site). Cluster site-awareness enhances key operations during the cluster lifecycle such as failover behavior, placement policies, heartbeat between the nodes, and quorum behavior. For more information, see [Site-aware Failover Clusters in Windows Server 2016](https://techcommunity.microsoft.com/t5/failover-clustering/site-aware-failover-clusters-in-windows-server-2016/ba-p/372060).
```
Comment (illfated) [inline WindowsServerDocs/failover-clustering/whats-new-in-failover-clustering.md:179]: End-of-line whitespace (blanks) removal and readability comma additions (2):
```suggestion
Windows Server 2016 includes site-aware failover clusters that enable group nodes in stretched clusters, based on their physical location (site). Cluster site-awareness enhances key operations during the cluster lifecycle, such as failover behavior, placement policies, heartbeat between the nodes, and quorum behavior. For more information, see [Site-aware Failover Clusters in Windows Server 2016](https://techcommunity.microsoft.com/t5/failover-clustering/site-aware-failover-clusters-in-windows-server-2016/ba-p/372060).
```

**Comment by illfated** [comment]
@RAJU2529 : Please commit these small improvement changes.

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Slightly off-topic : it took me a little while to understand where you got the name **John Marlin** from, but I realized that you are referring to the blog pages, not the MS Docs page.

- https://techcommunity.microsoft.com/t5/user/viewprofilepage/user-id/74167 (John Marlin)

cc: @JohnMarlin-MSFT

**Pull request merged**